express-postgres/server.ts

```diff
@@ -21,7 +21,7 @@ app.get('/api/films', async (req, res, next) => {
     `;
     const result = await db.query(sql);
     if (!result) {
-      throw new ClientError(404, 'films not found');
+      res.send([]);
     }
     res.send(result);
```

Comment:
This still isn't correct. How do you return the films that were read from the database?

Examine the response you are getting from this endpoint. You will see that it is an object, and not an array.

(Since the response is huge it may be easier to see the entire response by sending it to a file, with `http :8080/api/films > output.txt`.)

Reply:
I forgot to save before I added my files for my first change. I changed line 26 to `res.send(results.rows)` in my third commit.